var/spack/repos/builtin/packages/r/package.py

```diff
@@ -168,6 +171,9 @@ def configure_args(self):
         if '+memory_profiling' in spec:
             config_args.append('--enable-memory-profiling')
 
+        if '+libpng' in spec:
+            config_args.append('--with-libpng')
```

Comment:
Can you add an else clause that adds `--without-libpng`? Explicit is always better than implicit 😄